metrics/windows/disk.go

```diff
@@ -45,13 +46,13 @@ func NewDiskGenerator(interval time.Duration) (*DiskGenerator, error) {
 	for _, v := range drivebuf {
```

Comment:
According to https://msdn.microsoft.com/library/cc429332.aspx, `drivebuf` is a concatenation of null-terminated strings, so we should examine it as strings, not bytes.

Actually this code is not introduced by this PR, so we may fix it later.